README.md

```diff
@@ -87,6 +87,12 @@ module.exports.webpack = function(config) {
 };
 ```
 
+You can use this functionality to inject external styles or scripts if your
+components need e.g. Bootstrap to work.
+```js
+config.entry.push('../../../injectStyleTag.js');
```

Comment:
> Maybe you could use path.join(process.cwd(), 'injectStyleTag.js')

Duh, that will work :)

> So webpack doesn't support external files as entry points?

Not that I'm aware of. I tried to search for this, but couldn't find anything built in.